components/settings/SettingsColorMode.vue

```diff
@@ -31,6 +31,8 @@ const modes = [
     <button
       v-for="{ icon, label, mode } in modes"
       :key="mode"
+      data-glean="settings.interface.colorMode"
```

Comment:
Nice! I love the specificity here.

Reply:
Once these identifiers are finalized, can we please add them to the spreadsheet that I started?
https://docs.google.com/spreadsheets/d/1KX6TiyXXg2fE0a1IDKsy5O97ZrHYvjKohmIX_m8ECXY/edit#gid=1532914466